ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/auth/CompletionStreamAuthIT.scala

```diff
@@ -28,4 +28,12 @@ final class CompletionStreamAuthIT
       observer =>
         stub(CommandCompletionServiceGrpc.stub(channel), token).completionStream(request, observer)
 
+  // The completion stream is the one read-only endpoint where the application
+  // identifier has to be checked. Hence, we didn't put this test in a shared
+  // trait to no have to have to override the result for the transaction service
+  // authorization tests.
```

Comment:
```suggestion
  // The completion stream is the one read-only endpoint where the application
  // identifier is part of the request. Hence, we didn't put this test in a shared
  // trait to no have to have to override the result for the transaction service
  // authorization tests.
```
@SamirTalwar Does this clarify?

Reply:
Looks good to me.

Reply:
Implemented in https://github.com/digital-asset/daml/pull/6342/commits/afd6c193408fe34101bbe9f673d9eebab37291c3